src/main/play-doc/gettingStarted/Install.md

```diff
@@ -25,13 +25,15 @@ The requirements of a ConductR host are:
 * Python 3.4
 * Debian or Rpm package of ConductR
 
+RHEL/CentOS 7 support will be forthcoming.
```

Comment:
Perhaps quote the native packager in terms of Ubuntu only having partial support presently: https://github.com/sbt/sbt-native-packager#experimental-systemd-bootsystem-support